docs/bugs.json

```diff
@@ -1,4 +1,12 @@
 [
+    {
+        "name": "FreeFunctionAliasing",
+        "summary": "The compiler does not flag an error when two or more free functions with the same name and parameter types are defined in a source unit or when an imported free function alias shadows another free function with a different name but identical parameter types.",
+        "description": "Both definition of free functions with identical name and parameter types and an imported free function with an alias that shadows another function with a different name but identical parameter types were permitted due to which a call to either the multiply defined free function or the imported free function alias within a contract led to the execution of that free function which was defined first within the source unit. Subsequently defined identical free function definitions were silently ignored and their code generation was skipped.",
+        "introduced": "0.7.1",
+        "fixed": "0.7.2",
+        "severity": "medium"
```

Comment:
this is very easy to detect in tests and also not too likely to happen, so I would suggest `low`